e2e/dractions/actions.go

```diff
@@ -261,3 +263,15 @@ func waitAndUpdateDRPC(
 		return nil
 	})
 }
+
+// createNamespaceAndAnnotation is a helper function used during the protection of managed apps
+// to create namespaces and add annotations for VolSync based replication on both DR clusters
+// if the distribution is Kubernetes.
+// Returns an error if namespace creation or annotation fails.
+func createNamespaceAndAnnotation(ctx types.Context, appNamespace string, log *zap.SugaredLogger) error {
```

Comment:
The detail about the annotation is distro specific and we should not include it in the name of this helper. What we want is to have similar function names both managed and discovered appps:

    createNamespaces()
    createNamespacesForDisapp()

So we have a uniform structure of the code. Later we can check how to move the code so we can make this really uniform. For example if the deployer was implementing this we could have:

    deployer.CreateNamespaces()

And at some point we could use the same code to protect/unprotect any type of app. The differences would be in the deployer implemenation.